qutebrowser/utils/version.py

```diff
@@ -325,24 +332,38 @@ def is_outdated(self) -> Optional[bool]:
             return None
         return version < self.min_version
 
+    def __str__(self) -> str:
+        if not self.is_installed():
+            return f'{self.name}: no'
+        else:
```

Comment:
You can drop the `else:` since there's a `return` above ([eschew the extraneous else](https://youtu.be/JVVMMULwR4s?t=289) :wink:)